lib/models/device.coffee

```diff
@@ -51,6 +51,10 @@ getDeviceModel = (deps, opts) ->
 
 	exports = {}
 
+	# Infer dashboardUrl from apiUrl if former is undefined
+	if not dashboardUrl?
```

Comment:
Hm, interesting. We do actually have this logic in resin-settings-client: https://github.com/resin-io-modules/resin-settings-client/blob/master/lib/defaults.coffee#L73
OK let's merge this as is and update it when #310 is implemented.